doc/scope.hbs

```diff
@@ -0,0 +1,10 @@
+{{#if scope}}
```

Comment:
Are this templates required to support `@async` or it also contains some additional styling? Anyway, it would be nice to move all this jsdoc source to the separate NPM package, so we'll be able to share them across our projects.